ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -154,6 +155,14 @@ function ViolationsByPolicyCategoryChart({
         [searchFilter]
     );
 
+    // This widget uses a theme with the legend order in the opposite direction
+    // of the PatternFly defaults
+    const chartTheme = useMemo(() => {
+        const theme = cloneDeep(patternflySeverityTheme);
+        theme.legend.colorScale.reverse();
+        return theme;
+    }, []);
```

Comment:
I was wondering since it seems we only need to do this once (not dependent on re-renders or props), can this be defined outside the Component?

Reply:
Whoops, yeah good call. 😄